drivers/gles3/storage/texture_storage.cpp

```diff
@@ -2517,24 +2575,51 @@ void TextureStorage::render_target_set_override(RID p_render_target, RID p_color
 		rt->depth = cache->get().depth;
 		rt->size = cache->get().size;
 		rt->texture = p_color_texture;
-		return;
+		create_new_color_fbo = false;
+	}
+
+	uint32_t velocity_hash_key = hash_murmur3_one_64(p_velocity_texture.get_id());
+	velocity_hash_key = hash_murmur3_one_64(p_velocity_depth_texture.get_id(), velocity_hash_key);
+	velocity_hash_key = hash_fmix32(velocity_hash_key);
+
+	RBMap<uint32_t, GLuint>::Element *fbo;
+	if ((fbo = rt->overridden.velocity_fbo_cache.find(velocity_hash_key)) != nullptr) {
```

Comment:
```suggestion
	RBMap<uint32_t, GLuint>::Element *fbo = rt->overridden.velocity_fbo_cache.find(velocity_hash_key);
	if (fbo != nullptr) {
```
Prefer assignment at the declaration if possible for readability